warehouse/assemble/datawave/src/main/assembly/dist.xml

```diff
@@ -107,6 +107,8 @@
                 <include>${groupId}:datawave-ingest-nyctlc</include>
                 <include>${groupId}:datawave-ingest-json</include>
                 <include>gov.nsa.datawave.microservice:accumulo-utils</include>
+                <include>gov.nsa.datawave.microservice:base-rest-responses</include>
```

Comment:
All of the changes in this file need to go in `warehouse/assemble/datawave/pom.xml` as well--for RPM builds. Around line 534.

Reply:
good catch. thanks

Reply:
@brianloss Should be good now